src/encoded/schemas/file.json

```diff
@@ -236,21 +236,23 @@
         },
         "external_accession": {
             "title": "External accession",
-            "comment": "Only admins are allowed to set or update this value.",
+            "description": "For files that were not accessioned at our site but have an accession elsewhere.",
+            "comment": "Do not submit. Only admins are allowed to set or update this value.",
             "type": "string",
             "permission": "import_items",
             "uniqueKey": "accession"
         },
         "read_count": {
             "title": "Read count",
             "description": "Number of reads in fastq file.",
-            "comment": "Only admins are allowed to set or update this value.",
+            "comment": "Do not submit. This value is calculated and posted by checkfiles script upon submission.",
             "permission": "import_items",
             "type": "integer"
         },
         "fastq_signature": {
             "title": "Fastq file flowcell based signature",
             "description": "Fastq file flowcell based unique signature to reference a file.",
+            "comment": "Do not submit.  This value is calculated and posted by checkfiles script upon submission.",
```

Comment:
I wonder if we should use a word other than "calculated" here, as that has a specific meaning.

"This value is extracted from the fastq and posted by the checkfiles script"